contrib/local.mk.example

```diff
@@ -9,7 +9,7 @@
 
 # By default, the jemalloc family of memory allocation functions are used.
 # Uncomment the following to instead use libc memory allocation functions:
-# CMAKE_EXTRA_FLAGS="-DUSE_JEMALLOC=OFF"
+# DEPS_CMAKE_FLAGS := -DUSE_BUNDLED_JEMALLOC=OFF
```

Comment:
> we'd need USE_JEMALLOC={ON,OFF,AUTO} (with AUTO falling back to libc and ON failing if jemalloc not found)?

I don't think exposing an explicit AUTO option is reasonable. Some programs I know use what I consider to be a fairly straightforward model at configure-time.

For example, mpv has three general categories from what I've seen:
- Required modules: if they aren't automatically detected then the "configure" fails. Can't be disabled via an individual flag
- Optional wanted modules: default to being used if detected, but the configure doesn't fail if they're found. If the user explicitly requests such an optional module and it isn't found, the configure fails.
- Optional preferred modules: same as above, but don't default to being used if detected, so they require a flag.

What I'm getting at is this: would it be possible to have such an auto-detection scheme for jemalloc if USE_JEMALLOC isn't specified, but fall back on a more strict scheme if it is?